### ISSUE TITLE
Server Error: <class 'AttributeError'>, 'VirtualMachine' object has no attribute 'oob_ip_id'

### ISSUE THREAD

**Issue by lucafabbri365** [netbox-community/netbox#18259]
### Deployment Type

Self-hosted

### Triage priority

N/A

### NetBox Version

v4.1.8

### Python Version

3.10

### Steps to Reproduce

1. Create a Virtual Machine
2. Create an Interface and attach it to VM
3. Assign IP Address to the Interface
4. Try to Edit the IP Address => Server Error

### Expected Behavior

I expect to be able to edit the IP Address.

### Observed Behavior

```
Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'AttributeError'>

'VirtualMachine' object has no attribute 'oob_ip_id'

Python version: 3.10.12
NetBox version: 4.1.8
Plugins: None installed
```

**Comment by ITJamie** [comment]
duplicate: https://github.com/netbox-community/netbox/issues/18230

**Comment by bctiemann** [comment]
Also this is resolved in v4.1.9.
